src/medCore/database/medAbstractDatabaseImporter.cpp

```diff
@@ -202,89 +202,91 @@ void medAbstractDatabaseImporter::importFile ( void )
         currentFileNumber++;
 
         QFileInfo fileInfo ( file );
+        if (fileInfo.size() != 0)
```

Comment:
This ```if``` does not include all of the rest of this (way too long) function. You should follow the way this function is already coded the function and call the following code if the file is empty:
```
   emit showError (tr ( "The file is empty" ), 5000 );
   emit failure(this);
   return;
```